week-1/mandatory/2-classes-db/task.md

```diff
@@ -8,12 +8,30 @@ To submit this homework write the correct commands for each question here:
 
 ```sql
 1.select * from rooms where rate>=100.00;
-2.select * from reservations where extract(month from checkin_date)=extract(month from current_date) and checkout_date-checkin_date>3;
+2.select * from reservations where EXTRACT(MONTH from checkin_date)=EXTRACT(MONTH from current_date) and checkout_date-checkin_date>3;
 3.select * from customers where name LIKE 'm%';
 4.insert into room_types (room_type,def_rate) values ('PENTHOUSE',185.00);
- insert into rooms (room_no,room_type,rate) values (502,'PENTHOUSE',185.00);
- to check what is inserted
-select * from rooms order by room_no desc limit 10;
+5.insert into rooms (room_no,room_type,rate) values (502,'PENTHOUSE',185.00);
+5.insert into rooms (room_no,room_type,rate) values (502,'PENTHOUSE',185.00);
+to check what is inserted
+select * from rooms order by room_no DESC LIMIT 10;
+6. insert into rooms (room_no,rate,room_type) values(503 ,143.00,'PREMIER PLUS');
+7. select COUNT(*)  from reservations where EXTRACT(MONTH from checkin_date)=EXTRACT(MONTH from current_date -1);
```

Comment:
This only lists reservations with checkin dates in the current month, not the previous month and only works if we have less than a year's worth of reservations.